README.md

```diff
@@ -1,2 +1,13 @@
 # open-source-teams
 Place to document everything about our AnitaB.org Open Source teams
+
+Hey there!
+
+If you are here, it probably means you are either looking for a place to start contributing, or are looking for some direction regarding your current task. This is the place you will find our community's documentation common to all projects. Below is a list of documents currently in this repository. If you feel that none of this documentation is able to a help you, then you can reach out either by asking your question on zulip or by creating an issue requesting the information you need.
+
+## The documents in this repository:
```

Comment:
Since these 2 new documents are about documentation, I would just add this in a section called `Contributing` and encourage them to read the Code of Conduct and the reporting guidelines. Also when you mention this you can link the files in the text.